1.1.c

```diff
@@ -0,0 +1,34 @@
+#include<stdio.h>
+#include <conio.h>
+
+#define N 10
+
+void test()
+{
+    int numbers[N];
+    int i, j;
+
+    for (i = 0; i < N; i++)

```

Comment:
```suggestion
    for (int i = 0; i < N; i++)
```
Вот так можно объявить переменную в массиве.